core/src/main/java/org/infinispan/Cache.java

```diff
@@ -334,4 +334,13 @@
     */
    @Override
    void clear();
+
+   /**
+    * Performs a controlled shutdown of the cache. This method differs from {@link #stop()} only in clustered modes,
```

Comment:
It would be nice to know _how_ the method is different from `stop()` for clustered caches...